### PR TITLE
TU-467: Replaced Flash memory storage with RTC memory storage

### PR DESCRIPTION
**Pull request by MadBeignet**

### CODE REVIEW

File: SENDER/platformio.ini [hunk 1]
```diff
@@ -15,6 +15,7 @@ framework = arduino
 monitor_speed = 115200
 upload_speed = 460800
 upload_protocol = esptool
+board_build.flash_mode = dio
```
Comment (bennetthermanoff) [inline SENDER/platformio.ini]: leftover from debugging?

File: Mesh/src/measurement.h [hunk 2]
```diff
@@ -11,26 +11,34 @@
...
         Serial.println("Light Level: " + String(light_level));
         Serial.println("Battery Level: " + String(battery_level) + "mV");
     }
+    boolean notEmpty()
+    {
+        return measurement_num < num_measurements;
+    }
 };
 
+RTC_DATA_ATTR Measurement measurements[MAX_MEASUREMENTS]; // measurements stored in RTC memory
```
Comment (bennetthermanoff) [inline Mesh/src/measurement.h:41]: I'm assuming this allocates to the RTC memory

Comment (bennetthermanoff) [reply Mesh/src/measurement.h:41]: should test to see if we can read back after sleep

**Review by bennetthermanoff — APPROVED** [review]

File: Mesh/src/measurement.h (hunk 2 above)
Comment (MadBeignet) [reply Mesh/src/measurement.h:41]: we can

**Pull request merged**